.github/actions/create-release-candidate-v1/README.md

```diff
@@ -46,6 +46,23 @@ This action requires the following permission scopes:
 - `read:org` - To read the project board
 - `project` - access to project board
 
+## Release script requirements
+
+The `release-script-path` should point to a script which meets the following requirements:
+
+- May be invoked with either no arguments or a single argument (the string `patch`)
+   - in `patch` mode, version should bump to the next patch version
+   - in no-argument mode, version should bump to the appropriate next semver determined by commits since the previous version (via `lerna version`, `commit-and-tag-version`, or similar)
+- Should use `$GITHUB_REF` to determine a version suffix:
+   - if `$GITHUB_REF` is `refs/heads/develop`, append a `-next` version suffix (eg, `"-next." + process.env.GITHUB_SHA.substring(0, 8)` for NPM)
+   - if `$GITHUB_REF` is `refs/heads/main`, do not append a version suffix
+   - if `$GITHUB_REF` is anything else, exit with fail
+- Should update the version numbers in all files in the repo (`lerna.json`, `package.json`, `pom.xml`, etc.)
+   - This MUST include either a `lerna.json` or `package.json` at the root of the repo (even for non-js repos)
+- Should update CHANGELOG.md files
+- Should exit with 0 on success or non-zero on failure
+- No specific requirements on stdout/stderr format
```

Comment:
```suggestion
## Release script requirements

The `release-script-path` should point to a script which meets the following requirements:

- May be invoked with either no arguments or a single argument (the string `patch`)
   - in `patch` mode, version should bump to the next patch version
   - in no-argument mode, version should bump to the appropriate next semver determined by commits since the previous version (via `lerna version`, `commit-and-tag-version`, or similar)
- Should use `$GITHUB_REF` to determine a version suffix:
   - if `$GITHUB_REF` is `refs/heads/develop`, append a `-next` version suffix (eg, `"-next." + process.env.GITHUB_SHA.substring(0, 8)` for NPM)
   - if `$GITHUB_REF` is `refs/heads/main`, do not append a version suffix
   - if `$GITHUB_REF` is anything else, exit with fail
- Should update the version numbers in all files in the repo (`lerna.json`, `package.json`, `pom.xml`, etc.)
   - This MUST include either a `lerna.json` or `package.json` at the root of the repo (even for non-js repos)
- Should update CHANGELOG.md files
- Should exit with 0 on success or non-zero on failure
- No specific requirements on stdout/stderr format

An example of a [`prepare_release.sh` script](https://github.com/dequelabs/axe-core-npm/blob/develop/.github/scripts/prepare_release.sh).
```